[RU] Add `Contests` (#13923)
ppy/osu-wiki: wiki/Contests/ru.md

<!-- Please check redirects after adding new sections with year numbers. -->

# Конкурсы

Эта статья посвящена конкурсам, проводимым в сообществе osu!.

## Официальные конкурсы

> *Смотрите также: [Очки конкурсов](Contest_points), [Официальная поддержка конкурсов маппинга](Official_support)*

Reviewer: ```suggestion
*См. также: [Конкурсные очки](Contest_points), [Официальная поддержка конкурсов маппинга](Official_support)*
```
- https://osu.ppy.sh/wiki/ru/Article_styling_criteria/Formatting#%D1%81%D0%BC.-%D1%82%D0%B0%D0%BA%D0%B6%D0%B5
- считаю, что «конкурсные очки» звучит немного поорганичнее

| [Апрель 2021 (osu!catch)](https://osu.ppy.sh/community/contests/124) | [MYUKKE.](https://osu.ppy.sh/beatmaps/artists/121) | ![osu!catch][osu!catch] | ::{ flag=HK }:: [autofanboy](https://osu.ppy.sh/users/636114) | ::{ flag=TH }:: [rew0825](https://osu.ppy.sh/users/2488026) | *Н/П* | [#1](https://osu.ppy.sh/home/news/2021-06-25-results-monthly-beatmapping-contest-may-osu) |
| [May 2021 (osu!)](https://osu.ppy.sh/community/contests/127) | [Masahiro "Godspeed" Aoki](https://osu.ppy.sh/beatmaps/artists/136) | ![osu!][osu!] | ::{ flag=GB }:: [DeviousPanda](https://osu.ppy.sh/users/4966334) | ::{ flag=RU }:: [kuyusu](https://osu.ppy.sh/users/11758667) | ::{ flag=RU }:: [kuyusu](https://osu.ppy.sh/users/11758667) | [#1](https://osu.ppy.sh/home/news/2021-06-25-results-monthly-beatmapping-contest-may-osu) |

### [Aspire](Aspire)

> **Aspire** - уникальный конкурс маппинга, в котором участникам нужно было создать самую инновационную карту, без каких-либо ограничений [критериев ранкинга](/wiki/Ranking_criteria). Конкурс побуждает мапперов как можно лучше изучать технические возможности игры.

Reviewer: ```suggestion
**Aspire** - уникальный конкурс маппинга, в котором участникам нужно создать самую инновационную карту без оглядки на ограничения [критериев ранкинга](/wiki/Ranking_criteria). Aspire побуждает мапперов как можно лучше изучать технические возможности игры.
```
так немного покрасивее, кроме того, нет повторения

| :-: | :-- | :-- | :-- | :-- |
| [#1](/wiki/Contests/o!tFAC/1) | ::{ flag=KR }:: [woosungko](https://osu.ppy.sh/users/14184157) | ::{ flag=TH }:: [Ph0eNiiXZ](https://osu.ppy.sh/users/9463721) | ::{ flag=IT }:: [D3kuu](https://osu.ppy.sh/users/7807444) | [#1](https://osu.ppy.sh/community/forums/topics/1520800) |
| [#2](/wiki/Contests/o!tFAC/2) | ::{ flag=AR }:: [KyeX](https://osu.ppy.sh/users/15118934) | ::{ flag=KR }:: [woosungko](https://osu.ppy.sh/users/14184157) | ::{ flag=FI }:: [duski](https://osu.ppy.sh/users/6506484) | [#1](https://osu.ppy.sh/community/forums/topics/1588200) |
| [#3](/wiki/Contests/o!tFAC/3) | **Code:Red** (::{ flag=TH }:: [Ph0eNiiXZ](https://osu.ppy.sh/users/9463721), ::{ flag=TH }:: [Kusuhara Yui](https://osu.ppy.sh/users/9582525)) | **?XD** (::{ flag=SG }:: [arcpotato](https://osu.ppy.sh/users/12842392), ::{ flag=SG }:: [_gt](https://osu.ppy.sh/users/8301957)) | **:3** (::{ flag=KR }:: [woosungko](https://osu.ppy.sh/users/14184157), ::{ flag=GB }:: [Horiiizon](https://osu.ppy.sh/users/8071438)) | [#1](https://osu.ppy.sh/home/news/2022-12-26-results-osu!taiko-featured-artist-cup-3) |

### [osu! Mapping Olympiad](Mapping_Olympiad)

> **osu! Mapping Olympiad** - конкурс маппинга, проводящийся как духовный преемник ранних (до 2020 года) Monthly Beatmapping Contests. В этом конкурсе, участники должны были либо создать карту со всеми [сложностями](/wiki/Beatmap/Difficulty#уровни-сложности), начиная от Easy или Normal, либо определённое количество сложностей для каждой песни в каждой соответствующей итерации.

Reviewer: ```suggestion
**osu! Mapping Olympiad** - конкурс маппинга, являющийся духовным преемником ранних (до 2020 года) Monthly Beatmapping Contests. В зависимости от номера конкурса, участники должны создать либо карту со всеми [сложностями](/wiki/Beatmap/Difficulty#уровни-сложности), начиная от Easy или Normal, либо определённое количество сложностей для каждой песни.
```

> | Номер конкурса | Песня(и) | Режим игры | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | Результаты |

Reviewer: ```suggestion
| Номер конкурса | Песни | Режим игры | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] | Результаты |
```

| [#7](https://osu.ppy.sh/home/news/2020-06-06-osu-mapping-olympiad-7-mania) | [Frums - Star Of The COME ON!!](https://osu.ppy.sh/beatmaps/artists/68), [IAHN - Welcome (Original Mix)](https://osu.ppy.sh/beatmaps/artists/3), [PUP - DVP](https://osu.ppy.sh/beatmaps/artists/79), [HyuN - ENCOUNTER](https://osu.ppy.sh/beatmaps/artists/33), и [Cranky - Ran](https://osu.ppy.sh/beatmaps/artists/23) | ![osu!mania][osu!mania] | ::{ flag=US }:: [Toaph Daddy](https://osu.ppy.sh/users/7616811) и ::{ flag=KR }:: [Kawawa](https://osu.ppy.sh/users/4647754) | *Н/П* | ::{ flag=PH }:: [lenpai](https://osu.ppy.sh/users/5314573) и ::{ flag=CA }:: [BringoBrango](https://osu.ppy.sh/users/10274043) | [#1](https://osu.ppy.sh/home/news/2020-12-01-osu-mapping-olympiad-7-concludes) |

### Community Mapping Contest

> **Community Mapping Contest**  - конкурс маппинга, в котором участникам можно присылать карту любой сложности в любом режиме игры, после чего все сложности победителей среди всех сложностей и режимов соединяются в одну карту.

Reviewer: ```suggestion
**Community Mapping Contest**  - конкурс маппинга, в котором участникам можно делать любую сложность в любом режиме игры. В конечном счёте все сложности, победившие в каждом режиме игры, объединяются в одну карту.
```
так более понятно и просто, мне кажется

| [#3.2 (osu!taiko)](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns) | [\*namirin - closing eyes](https://osu.ppy.sh/beatmaps/artists/18) | ![osu!taiko][osu!taiko] | ::{ flag=HK }:: [Nardoxyribonucleic](https://osu.ppy.sh/users/876419) | ::{ flag=US }:: [Jonarwhal](https://osu.ppy.sh/users/3653035) | *Н/П* | [#1](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |
| [#3.3](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns) | [Fractal Dreamers - Fortuna Redux](https://osu.ppy.sh/beatmaps/artists/15) | ![osu!][osu!] | ::{ flag=FI }:: [DTM9 Nowa](https://osu.ppy.sh/users/5428909) | ::{ flag=MY }:: [HootOwlStar](https://osu.ppy.sh/users/4341302) | *Н/П* | [#1](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |

### [A Labour of Love](A_Labour_of_Love_Contest)

> **A Labour of Love** - командный конкурс, охватывающий несколько художественных дисциплин, в котором командам даётся задание - создать ремикс/аранжировку песни, хорошо известной в сообществе osu!, создать карту из этого ремикса и сделать оригинальный сториборд/фоновое видео как визуальное дополнение к этой карте. Как [указано в этом твите](https://twitter.com/ppy/status/1333362145224843264), конкурс проводится по образцу [ивентов THE BMS OF FIGHTERS](https://www.bmsoffighters.net/) и подобных ему соревнований.

Reviewer: ```suggestion
**A Labour of Love** - командный конкурс, охватывающий несколько художественных дисциплин, в котором командам даётся задание: создать ремикс или аранжировку песни, хорошо известной в сообществе osu!, сделать карту на этот ремикс и дополнить её оригинальным сторибордом или фоновым видео. Как указано в [этом твите](https://twitter.com/ppy/status/1333362145224843264), конкурс проводится по образцу [ивентов THE BMS OF FIGHTERS](https://www.bmsoffighters.net/) и подобных соревнований.
```
мини-сокращение идёт на пользу, лучше читается теперь

> | Номер конкурса | Ремикс песни | ![Gold crown][GCrown] (Победитель) | ![Silver crown][SCrown] (Второе место) | Победитель по мнению сообщества | Результаты |

Reviewer: ```suggestion
| Номер конкурса | Песня для ремикса | ![Gold crown][GCrown] (Победитель) | ![Silver crown][SCrown] (Второе место) | Победитель по мнению сообщества | Результаты |
```

| :-: | :-- | :-- | :-- | :-- | :-- |
| [1](https://osu.ppy.sh/home/news/2020-11-30-a-labour-of-love) | [Kuba Oms - My Love](https://osu.ppy.sh/beatmaps/artists/13) | extremely valid (::{ flag=US }:: [BilliumMoto](https://osu.ppy.sh/users/3862471), ::{ flag=FR }:: [Shinwoir](https://osu.ppy.sh/users/8984574), ::{ flag=DE }:: [Mao](https://osu.ppy.sh/users/2204515), ::{ flag=FR }:: [Realazy](https://osu.ppy.sh/users/918297), и ::{ flag=GB }:: [DeviousPanda](https://osu.ppy.sh/users/4966334)) | schoolboy fans (::{ flag=RU }:: [Shadren](https://osu.ppy.sh/users/745494), ::{ flag=UA }:: [wajinshu](https://osu.ppy.sh/users/6339790), ::{ flag=RU }:: [Shmiklak](https://osu.ppy.sh/users/5504231), и ::{ flag=RU }:: [Daycore](https://osu.ppy.sh/users/5596337)) | Team Red but Blue (::{ flag=CA }:: [Redside](https://osu.ppy.sh/users/16039046), ::{ flag=US }:: [Tofumang](https://osu.ppy.sh/users/4817223), ::{ flag=CA }:: [Chompy](https://osu.ppy.sh/users/7427035), ::{ flag=US }:: [Noffy](https://osu.ppy.sh/users/1541323), и ::{ flag=CA }:: [Zer0-G](https://osu.ppy.sh/users/12577911)) | [#1](https://osu.ppy.sh/home/news/2021-04-27-results-a-labour-of-love) |

### Best of/Community Choice

> **Best of/Community Choice** - конкурс с публичным голосованием, который регулярно проводится для выяснения самых популярных карт <!-- https://osu.ppy.sh/home/news/2021-02-05-community-choice-2020-results --> в глазах сообщества osu! в каждом году.

Reviewer: ```suggestion
**Best of/Community Choice** - ежегодный конкурс с публичным голосованием, который проводится для выяснения самых популярных карт <!-- https://osu.ppy.sh/home/news/2021-02-05-community-choice-2020-results --> по мнению сообщества osu!.
```

| [#40](https://osu.ppy.sh/home/news/2024-06-14-aerial-antics-art-contest) | Aerial Antics Art Contest | Маскоты osu! занимаются небесными делами | ::{ flag=CA }:: [RedcXca](https://osu.ppy.sh/users/14056601) | ::{ flag=TR }:: [mocapixel](https://osu.ppy.sh/users/11596939) | ::{ flag=ID }:: [KAITO thighs](https://osu.ppy.sh/users/13103233) | [#1](https://osu.ppy.sh/home/news/2024-07-15-aerial-antics-art-results), [#2](https://osu.ppy.sh/community/contests/215) |
| [#41](https://osu.ppy.sh/home/news/2024-09-06-autumn-fanart-contest) | Autumn 2024 Fanart Contest | Осень | ::{ flag=ID }:: [KAITO thighs](https://osu.ppy.sh/users/13103233) | ::{ flag=CA }:: [leeian](https://osu.ppy.sh/users/12595640) | ::{ flag=SG }:: [Type](https://osu.ppy.sh/users/7520534) | [#1](https://osu.ppy.sh/home/news/2024-10-09-autumn-2024-fanart-results), [#2](https://osu.ppy.sh/community/contests/221) |

> | [#42](https://osu.ppy.sh/home/news/2024-12-03-midnight-moment-art-contest) | Midnight Moment Art Contest | Маскоты osu! разделяют момент зимним вечером | ::{ flag=CA }:: [leeian](https://osu.ppy.sh/users/12595640) | ::{ flag=ID }:: [KAITO thighs](https://osu.ppy.sh/users/13103233) | ::{ flag=RU }:: [LeeNarie](https://osu.ppy.sh/users/2667849) | [#1](https://osu.ppy.sh/home/news/2025-01-01-midnight-moment-art-contest-results), [#2](https://osu.ppy.sh/community/contests/226) |

Reviewer: ```suggestion
| [#42](https://osu.ppy.sh/home/news/2024-12-03-midnight-moment-art-contest) | Midnight Moment Art Contest | Маскоты osu! зимним вечером | ::{ flag=CA }:: [leeian](https://osu.ppy.sh/users/12595640) | ::{ flag=ID }:: [KAITO thighs](https://osu.ppy.sh/users/13103233) | ::{ flag=RU }:: [LeeNarie](https://osu.ppy.sh/users/2667849) | [#1](https://osu.ppy.sh/home/news/2025-01-01-midnight-moment-art-contest-results), [#2](https://osu.ppy.sh/community/contests/226) |
```
мне кажется, «разделяют момент» по-русски никто не говорит

### Locus

> **Locus 2025** - многослойный конкурс, начавшийся в начале 2025 года, созданный для сплочения креативных кружков по osu! вместе, в котором участники должны создать абсолютно новые карты с музыкой и визуалом.

Reviewer: ```suggestion
**Locus 2025** - многослойный конкурс, начавшийся в начале 2025 года и созданный для сплочения креативных кружков по osu!. Его участники должны создать совершенно новые карты с музыкой и визуалом.
```
так получше и менее нагруженно читается

| Год | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] | Результаты |
| :-: | :-- | :-- | :-- | :-- |
| [2025](/wiki/Contests/Locus/2025) | *Будет определено* | *Будет определено* | *Будет определено* | *Будет определено* |

### osu! Beatmap Blueprints Contest

> *osu! Beatmap Blueprints Contest* - одноразовый конкурс, проводимый в ознаменование запуска [наклеек Beatmap Blueprints](https://osu.ppy.sh/store/products/9) в [магазине osu!](https://osu.ppy.sh/store/listing). В этом конкурсе, участники соревновались в создании лучшей наклейки карты в любом визуальном формате, используя все ресурсы, предоставленные в пакете мерча Beatmap Blueprints.

Reviewer: ```suggestion
*osu! Beatmap Blueprints Contest* - одноразовый конкурс в честь выпуска [наклеек Beatmap Blueprints](https://osu.ppy.sh/store/products/9) в [магазине osu!](https://osu.ppy.sh/store/listing). Участники соревнуются в создании лучшей наклейки карты в любом визуальном формате, используя ресурсы, предоставленные в пакете мерча Beatmap Blueprints.
```

| Номер конкурса | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] | Результаты |
| :-: | :-- | :-- | :-- | :-- |

> | [#1](https://osu.ppy.sh/home/news/2015-03-20-osu-beatmap-blueprints-available-contest) | *Н/П* | *Н/П* | *Н/П* | *Н/П* |

Reviewer: лучше оставить N/A, это универсальное сокращение (а Н/П, как по мне, звучит очень н/п)

### osu!idol

> **osu!idol** - конкурс певцов, в котором участники со всего сообщества соревнуются в создании лучшего кавера (в одиночку либо с партнёром) среди множества стадий оценки и голосования сообщества. Конкурс назван в честь [франшизы Idols](https://ru.wikipedia.org/wiki/Idol_(телепроект)) и в основном вдохновлен различными ТВ-конкурсами пения.

Reviewer: ```suggestion
**osu!idol** - конкурс пения, в котором участники соревнуются в исполнении лучшего кавера (в одиночку либо с партнёром) и участвуют во множестве стадий оценки и голосования сообщества. Конкурс назван в честь [франшизы Idols](https://ru.wikipedia.org/wiki/Idol_(телепроект)) и в основном вдохновлен различными ТВ-конкурсами пения.
```

| [#5 (2018)](https://osu.ppy.sh/home/news/2018-07-01-osu-idol-2018-auditions-now-open) | ::{ flag=US }:: [Will Stetson](https://osu.ppy.sh/users/4909088) | ::{ flag=ID }:: [Zeva\[FL\]](https://osu.ppy.sh/users/2805170) | ::{ flag=US }:: [MjIsMaster](https://osu.ppy.sh/users/9922896) | [Will Stetson - Snow Drive](https://assets.ppy.sh/contests/71/entries/Snow%20Drive.mp3) | [#1](https://osu.ppy.sh/home/news/2018-11-05-osu-idol-2018-finals-community-voting-results) |
| [#6 (2019)](https://osu.ppy.sh/home/news/2019-07-02-osu-idol-2019-auditions-now-open) | ::{ flag=PH }:: [iamALieee](https://osu.ppy.sh/users/10093980) | ::{ flag=US }:: [aceredshirt13](https://osu.ppy.sh/users/13736238) | ::{ flag=US }:: [moof](https://osu.ppy.sh/users/8457202) | [iamALieee - White First Love](https://assets.ppy.sh/contests/83/entries/osu!idol%202019%20-%20iamALieee%20-%20White%20First%20Love.mp3) | [#1](https://osu.ppy.sh/home/news/2019-12-21-osu-idol-2019-finals-community-voting-results) |
| [#7 (2020)](https://osu.ppy.sh/home/news/2020-09-01-osu-idol-2020-auditions-now-open) | *Н/П* | *Н/П* | *Н/П* | *Н/П* | *Н/П* |

### osu!remix

> **osu!remix** - конкурс ремиксов, в котором музыканты со всего сообщества должны создать лучшей ремикс выбранной песни, которая хорошо известна благодаря сообществу osu! в том или ином виде. Это первый конкурс, который проводился через страницу [списка конкурсов](https://osu.ppy.sh/community/contests).

Reviewer: ```suggestion
**osu!remix** - конкурс, в котором музыканты должны сделать лучший ремикс заранее выбранной песни, которая хорошо известна сообществу osu!. Это первый конкурс, который проводился через страницу [списка конкурсов](https://osu.ppy.sh/community/contests).
```
касательно конкурса, который «проводился через страницу» — я понимаю, что имеется в виду, но, как по мне, не очень изящно написано, может, тут лучше подойдёт что-нибудь вроде:

```md
Это первый конкурс, который проводился [полностью на сайте osu!](https://osu.ppy.sh/community/contests).
```

| Номер конкурса | Ремикс песни | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] | Результаты |
| :-: | :-- | :-- | :-- | :-- | :-- |
| [#1](https://osu.ppy.sh/home/news/2016-05-06-osu-circles-remix-contest) | [nekodex - circles!](https://osu.ppy.sh/beatmaps/artists/1) | ::{ flag=US }:: [BilliumMoto](https://osu.ppy.sh/users/3862471) | ::{ flag=FR }:: [Kurokotei](https://osu.ppy.sh/users/398275) | ::{ flag=NL }:: [happy30](https://osu.ppy.sh/users/27767) | [#1](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) |

### [Skinning Contest](Skinning_Contest)

> **Skinning Contest**- конкурс скинов, в котором креаторы со всего сообщества соревнуются в создании лучшего скина за ограниченное время.

Reviewer: ```suggestion
**Skinning Contest**- конкурс скинов, в котором мастера со всего сообщества соревнуются в создании лучшего скина за ограниченное время.
```

практически «Поколение П», но наоборот — никаких *креаторов*, давай нормальные слова!

| [#2](/wiki/Contests/osu!trivium_Quiz/2021) | Squid balls (::{ flag=MY }:: [squidstain](https://osu.ppy.sh/users/11073207), ::{ flag=US }:: [Codazen](https://osu.ppy.sh/users/18291144), ::{ flag=HK }:: [autofanboy](https://osu.ppy.sh/users/636114), и ::{ flag=MY }:: [cdh](https://osu.ppy.sh/users/2403621)) | untouchaball-epic-fail-crew-9000-nightmarecore-mix (::{ flag=US }:: [1103](https://osu.ppy.sh/users/7306698), ::{ flag=US }:: [GalacticLightN](https://osu.ppy.sh/users/4772172), ::{ flag=AU }:: [Beat43210](https://osu.ppy.sh/users/5664171), и ::{ flag=US }:: [Backfire](https://osu.ppy.sh/users/263110)) | sleep squad (::{ flag=KR }:: [Civil oath](https://osu.ppy.sh/users/3216107), ::{ flag=SE }:: [PuffBuck](https://osu.ppy.sh/users/4234525), ::{ flag=NZ }:: [deadbeat](https://osu.ppy.sh/users/128370), и ::{ flag=US }:: [ztrot](https://osu.ppy.sh/users/6347)) | [#1](https://www.twitch.tv/videos/1237832029) |

## Конкурсы сообщества

*Неофцициальные конкурсы* - конкурсы, проводящиеся участниками сообщества. 

### [Mapping to Pander: Taiko](M2P)

> **Mapping to Pander: Taiko** - серия конкурсов по маппингу в osu!taiko с двумя основными особенностями: здесь есть только один судья за итерацию, и участники могут сделать карту под любую песню.

Reviewer: ```suggestion
**Mapping to Pander: Taiko** - серия конкурсов по маппингу в osu!taiko с двумя главными особенностями: наличие одного-единственного судьи и абсолютная свобода в выборе песни.
```

> Цель конкурса проста: участникам нужно создать карту, которая больше всего понравится судье. Судьям не дают никаких критериев; они сами решают, как они оценивают карты, либо создавая свои критерии, либо просто оценивая карты от лучшей к худшей.

Reviewer: ```suggestion
Цель конкурса проста: участникам нужно создать карту, которая больше всего понравится судье. Судьям не дают никаких критериев: они сами решают, как оценивать публикации, либо создавая свои критерии, либо просто сортируя карты по качеству.
```
чуть перефразировал, чтобы получше читалось
